sap_business_document_processing/document_information_extraction_client/dox_api_client.py

```diff
@@ -0,0 +1,608 @@
+        "state":"", "email":"", "phone":"", "bankAccount":"", "taxId":""}
+        :return: The API endpoint response as dictionary
+        """
+        return self.upload_enrichment_data(client_id, DoxDataType.BUSINESS_ENTITY, data, DoxDataSubType.CUSTOMER)
+
+    def get_enrichment_data(self, client_id, data_type, data_id=None, offset=None, limit=None, subtype=None,
+                            system=None, company_code=None):
+        """
+        Gets the enrichment data records filtered by the provided parameters
+        :param client_id: The ID of the client for which the enrichment data was created
+        :param data_type: The type of the data records. Can be given as an instance of :enum:`DoxDataType` or string.
+        Options are 'businessEntity' or 'employee'
+        :param data_id: (optional) The ID of a single data record. Only one will be returned
+        :param offset: (optional) The index of the first record to be returned
+        :param limit: (optional) The maximum number records to be returned
+        :param subtype: (optional) The subtype of the records. Only used for type 'businessEntity'. Can be given as
+        instance of :enum:`DoxDataSubType` or as string. Options are 'supplier', 'customer' or 'companyCode'
+        :param system: (optional) The system of a single record
+        :param company_code: (optional) The company code of a single record
+        :return: A list of enrichment data records. Returns a list with one item when data_id is given
+        """
+        params = {
+            API_FIELD_CLIENT_ID: client_id,
+            API_REQUEST_FIELD_ENRICHMENT_TYPE: str(data_type)
+        }
+        if data_id is not None:
+            params[API_REQUEST_FIELD_ENRICHMENT_ID] = data_id
+        if offset is not None:
+            params[API_REQUEST_FIELD_OFFSET] = offset
+        if limit is not None:
+            params[API_REQUEST_FIELD_LIMIT] = limit
+        if subtype is not None:
+            params[API_REQUEST_FIELD_ENRICHMENT_SUBTYPE] = str(subtype)
+        if system is not None:
+            params[API_REQUEST_FIELD_ENRICHMENT_SYSTEM] = system
+        if company_code is not None:
+            params[API_REQUEST_FIELD_ENRICHMENT_COMPANYCODE] = company_code
+
+        response = self.get(DATA_ENDPOINT, params=params,
+                            log_msg_before=f'Getting enrichment data records for client {client_id}')
+        self.logger.info(f'Successfully got {len(response.json()[API_FIELD_VALUE])} enrichment data records')
+        return response.json()[API_FIELD_VALUE]
+
+    def delete_all_enrichment_data(self, data_type=None):
+        """
+        This endpoint is deleting all master data records for the account
+        :param data_type: (Optional) The type of enrichment data that should be deleted. Can be given as
+        :enum:`DoxDataType` or string. Options are 'businessEntity' or 'employee'
```

Comment:
Let's refer to the documentation instead of explicitly mentioning `businessEntity` and `employee`

Reply:
done